Sprint7/TemăJS.js

```diff
@@ -0,0 +1,45 @@
+// 1. Returnează un boolean dacă un număr este divizibil cu 10
+function divizibil(num){
+  if (num % 10 === 0){
+    console.log(`${num} este divizibil cu 10!`)
+  } else {
+    console.log(`${num} nu este divizibil cu10!`)
+  }
+}
+
+
+// 2.Scrieți o funcție numită tellFortune
+function tellFortune(numarCopii,numePartener,locatieGeografica,locMunca){
+  console.log(`Vei fi un ${locMunca} în ${locatieGeografica} căsătorit cu ${numePartener} și vei avea ${numarCopii}`)
```

Comment:
```suggestion
  console.log(`Vei fi un ${locMunca} în ${locatieGeografica} căsătorit cu ${numePartener} și vei avea ${numarCopii}`);
```
(opt) fiecare instructiune se termina cu `;`